src/structure/structure.ts

```diff
@@ -292,6 +296,34 @@ export class StructureViewer {
         }
     }
 
+    /*
+     * Removes a widget from the structure viewer grid.
+     * The parameter force pertains to removing the *only* widget in the grid,
+     * which should only be done when changing datasets.
```

Comment:
Since we are no longer changing datasets, should we remove this `force` parameter?